### PR TITLE
Changing sentence in UI that could suggest a private key is not 100% private

### PR DESCRIPTION
**Pull request by jadehamel**
Well, I did not ask the community before about this change, but I strongly believe that there are many ways to share funds or securities without sharing a pk with anyone. I believe a pk must be own and knows by 1 entity only.... thank you for your comments! :)

### CODE REVIEW

**Comment by patricklodder** [comment]
I agree with what you're trying to achieve here (and you don't need to ask anyone to make a pull request here, so good job!)

To make this acceptable for merge, could you please remove the edits to the locale files (`src/qt/locale/*.ts`)? We only need the change in the .cpp files as the locale files have automation (that was refined in #2870) to make this appear/invalidate for all translations at once. I'm anyway going to add some other edits that I ran into earlier today, so we can reduce work for translators by doing that all at once. 

If you're not sure how to revert the edits, just say so and I can do it for you as long as you allow edits by maintainers (on the bottom of the sidebar 👉)

**Comment by jadehamel** [comment]
> 

Hell

> I agree with what you're trying to achieve here (and you don't need to ask anyone to make a pull request here, so good job!)
> 
> To make this acceptable for merge, could you please remove the edits to the locale files (`src/qt/locale/*.ts`)? We only need the change in the .cpp files as the locale files have automation (that was refined in #2870) to make this appear/invalidate for all translations at once. I'm anyway going to add some other edits that I ran into earlier today, so we can reduce work for translators by doing that all at once.
> 
> If you're not sure how to revert the edits, just say so and I can do it for you as long as you allow edits by maintainers (on the bottom of the sidebar 👉)

OK! thank you very much for your reply! I understand that working with .po files before! I'll change it!

**Comment by jadehamel** [comment]
> > 
> 
> Hell
> 
> > I agree with what you're trying to achieve here (and you don't need to ask anyone to make a pull request here, so good job!)
> > To make this acceptable for merge, could you please remove the edits to the locale files (`src/qt/locale/*.ts`)? We only need the change in the .cpp files as the locale files have automation (that was refined in #2870) to make this appear/invalidate for all translations at once. I'm anyway going to add some other edits that I ran into earlier today, so we can reduce work for translators by doing that all at once.
> > If you're not sure how to revert the edits, just say so and I can do it for you as long as you allow edits by maintainers (on the bottom of the sidebar 👉)
> 
> OK! thank you very much for your reply! I understand that working with .po files before! I'll change it!

Did I did it right? Otherwise do it :) ... please:)

**Comment by patricklodder** [comment]
> Did I did it right? Otherwise do it :) ... please:)

It went well. We will need to do a rebase & squash to make it clean, but can do that later.

For now: @alamshafil, since you coded the original but I cannot seem to find the reason why we changed from "with anyone" to "with an untrustworthy person" - do you agree with the change? Same question to @DimiFW.

**Comment by jadehamel** [comment]
> > Did I did it right? Otherwise do it :) ... please:)
> 
> It went well. We will need to do a rebase & squash to make it clean, but can do that later.
> 
> For now: @alamshafil, since you coded the original but I cannot seem to find the reason why we changed from "with anyone" to "with an untrustworthy person" - do you agree with the change? Same question to @DimiFW.

For the termination "using: 'with anyone'", I was inspired with line 156 from src/qt/overviewpage.cpp

File: src/qt/overviewpage.cpp [hunk 1]
```diff
@@ -159,8 +159,8 @@ void OverviewPage::UpdateTip()
         tr("Make sure to keep your wallet updated."),
         tr("Backup your private key to recover your coins, using 'File' > 'Backup Wallet'"),
         tr("Always do your own research before using an external cryptocurrency service"),
-        tr("Never share your private key to an untrustworthy person."),
-        tr("Who owns the private keys owns the coins."),
+        tr("Never share your private key with anyone"),
+        tr("Who own the private keys own the coins."),
```
Comment (chromatic) [inline src/qt/overviewpage.cpp:163]: Minor nit: I think the first `own` should remain `owns`. The change to the second looks correct.

Maybe "Whoever owns the private keys owns the coins." would be even better.

**Comment by alamshafil** [comment]
> For now: @alamshafil, since you coded the original but I cannot seem to find the reason why we changed from "with anyone" to "with an untrustworthy person" - do you agree with the change? Same question to @DimiFW.

I agree with the change, it makes sense to me. I can't remember where I got "with an untrustworthy person" from; I must have copied and pasted it.

**Comment by chromatic** [comment]
This looks pretty close to me. @jadehamel can you squash these commits into a single change? I'm happy to help if you'd like it.

Otherwise I still think "Whoever owns" is more idiomatic English for the second phrasing.

**Comment by patricklodder** [comment]
> @jadehamel can you squash these commits into a single change? I'm happy to help if you'd like it.

I squashed it for you per [this request](https://github.com/dogecoin/dogecoin/pull/2953#issuecomment-1125628360), which also removed the [merge issue](https://github.com/dogecoin/dogecoin/pull/2953#discussion_r872568976) @chromatic mentioned.

**Review by patricklodder — APPROVED** [review]
ACK from me. Will stage a couple more textual fixes and update translation files with the few changes we have.

@chromatic could you please let us know if we can have your stamp on this too?

**Review by chromatic — APPROVED** [review]
ACK from me too. It's an improvement I'm glad to see.

**Pull request merged**